playwright.config.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="./node_modules/@types/node/index.d.ts" />
+import "dotenv/config";
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "./e2e",
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: "html",
```

Comment:
Kristof mentioned junit output can be formatted nicely. SO we could change the output if needed later